workflows/src/main/java/de/uniks/backend/WorkflowsBackend.java

```diff
@@ -0,0 +1,12 @@
+package de.uniks.backend;
```

Comment:
Please use org.fulib - maybe org.fulib.workflows.webapp?

Reply:
Good suggestion and also refactored